src/main/java/com/iexec/core/chain/BlockchainConnectionHealthIndicator.java

```diff
@@ -0,0 +1,165 @@
+package com.iexec.core.chain;
```

Comment:
The license is missing on this new file

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/601/commits/befbca97c9f719e79b6f324eb72e169bf55c200b.